components/org.wso2.carbon.identity.application.authenticator.fido2.endpoint/src/gen/java/org/wso2/carbon/identity/application/authenticator/fido2/endpoint/FinishRegistrationApi.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.identity.application.authenticator.fido2.endpoint;
+
+import org.wso2.carbon.identity.application.authenticator.fido2.endpoint.dto.*;
```

Comment:
Avoid * imports, check this in all the classes unless autogenerated class.

Reply:
This is an auto-generated class.